contributing_to_docs/doc_guidelines.adoc

```diff
@@ -743,21 +743,59 @@ Replace all public IP addresses with an address from the following blocks. These
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
 ====
 
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+Using a lead-in link sentence that adheres to guidelines maintains consistency across OpenShift Container Platform documentation.
+
+For situations where you cannot use the `xref` attribute, such as referencing a module within a module, ensure that you adhere to the following guidance for lead-in link sentences.
+
+* Use the following format when pointing to a section in the same document: `For more information, see "<section_name>."`
+* Wrap the `<section_name>` string in quotation marks. The section name can be the title of a module or an assembly.
+* Do not state "in the Additional Resources section" or other variants. Mentioning the `<section_name>` is enough to prompt a documentation user to find the section in the same document.
+
+For situations where you can use the `xref` attribute, adhere to the following guidance:
+
+* When pointing to a section that exists in the same documentation set, create an _Additional resources_ section in the assembly file and then place the xref as a bulleted item under this section. For example:
++
+[source,text]
+----
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc#installation-config-parameters-vsphere[Installation configuration parameters]
+----
+* When pointing to a link in a section that exists in another customer product document, use the `For more information on <topic>, see "<section_name>" in _<document_name>_.` format. Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
+
+For situations where you can use the `link` attribute, adhere to the following guidance:
+
+* When pointing to a link that exists external to Red Hat's customer portal documentation, use the `link` attribute, place the link in an _Additional resources_ section, and use the `<document_heading_name> (<document_source>)` format. For example, see link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] (AWS documentation).
+
+[NOTE]
+====
+Certain Red Hat resources, such as Knowledgebase articles, exist in a separate server location to Red Hat customer portal documentation. For these resources, use the `link` attribute format.
+
+For more information, see link:https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links[Links to Red Hat Knowledgebase articles] (Red Hat supplementary style guide for product documentation).
+====
+
+* For more information about the differences between the _Additional resources_ and _Next steps_ headings, see xref:additional-resources-sections[Additional resources sections].
+* For more information about the `xref` and `link` attributes, see xref:links-conditionals-assemblies[Links, hyperlinks, and cross references].
```

Comment:
Maybe put this in both the xref and the link section?

```
* For more information about using `xref`, see xref:links-conditionals-assemblies[Links, hyperlinks, and cross references].
```
in the xref section. and in this link section:
```
* For more information about using `link`, see xref:links-conditionals-assemblies[Links, hyperlinks, and cross references].
```